mono/utils/mono-error.c

```diff
@@ -611,6 +623,10 @@ mono_error_prepare_exception (MonoError *oerror, MonoError *error_out)
 		exception = mono_get_exception_argument (error->type_name, mono_internal_error_get_message (error));
 		break;
 
+	case MONO_ERROR_ARGUMENT_NULL:
```

Comment:
I based this code on how `MONO_ERROR_ARGUMENT` is handled, and it is not handled neither in `mono_error_set_from_loader_error` neither in `mono_loader_set_error_from_mono_error`. So is it something we need to add for both `MONO_ERROR_ARGUMENT` and `MONO_ERROR_ARGUMENT_NULL`, or none of them?